src/wing/CCPACSWing.h

```diff
@@ -64,125 +77,250 @@ friend class CTiglWingBuilder;
     TIGL_EXPORT void SetSymmetryAxis(const TiglSymmetryAxis& axis) override;
     TIGL_EXPORT void SetParentUID(const boost::optional<std::string>& value) override;
 
-    // Returns whether this wing is a rotor blade
+    /**
+     * @brief Returns whether this wing is a rotor blade
+     * @return bool
+     */
     TIGL_EXPORT bool IsRotorBlade() const;
 
-    // Returns the parent configuration
+    /**
+     *  @brief Returns the parent configuration
+     *  @return CCPACSConfiguration&
+     */
     TIGL_EXPORT CCPACSConfiguration & GetConfiguration() const;
 
-    // Get section count
+    /**
+     * @brief Get section count
+     * @return int
+     */
     TIGL_EXPORT int GetSectionCount() const;
 
-    // Returns the section for a given index
+    /**
+     * @brief Returns the section for a given index
+     * @return CCPACSWingSection&
+     */
     TIGL_EXPORT CCPACSWingSection& GetSection(int index);
     TIGL_EXPORT const CCPACSWingSection& GetSection(int index) const;
 
-    // Getter of the number of segments of the wing
+
+    /**
+     *  @brief Returns the number of segments of the wing
+     *  @param index
+     *  @return int
+     */
     TIGL_EXPORT int GetSegmentCount() const;
 
-    // Returns the segment for a given index or uid
+    /**
+     * @brief Returns the segment for a given index or uid
+     * @param index Segment index
+     * @param uid Segment UID
+     * @return CCPACSWingSegment&
+     */
     TIGL_EXPORT CCPACSWingSegment& GetSegment(const int index);
     TIGL_EXPORT const CCPACSWingSegment& GetSegment(const int index) const;
     TIGL_EXPORT CCPACSWingSegment& GetSegment(std::string uid);
     TIGL_EXPORT const CCPACSWingSegment& GetSegment(std::string uid) const;
 
-    // Getter of the number of component segments of the wing
+    /**
+     * @brief Returns the number of component segments of the wing
+     * @return int
+     */
     TIGL_EXPORT int GetComponentSegmentCount() const;
 
-    // Returns the segment for a given index or uid
+    /**
+     * @brief Returns the segment for a given index or uid
```

Comment:
See above: I believe we need one docsstring for each function, so that the documentation is rendered for every function in the generated html doc.